apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Elements.purs

```diff
@@ -334,6 +363,21 @@ subscriptionUpdates self@{ props: props@{ now, subscription: sub@{ subsno, packa
                     ]
         }
 
+renewSubscriptionComponent :: Types.Self -> JSX
+renewSubscriptionComponent self@{props, state} =
+  props.renewSubscription
+    { subscription: props.subscription
+    , hasAddress: isJust $ toMaybe $ self.props.user.address
+    , onCancel: do
+        self.props.setRenewingSubscription Nothing
+        self.setState _ { wrapperProgress = AsyncWrapper.Ready }
+    , onSuccess: do
+        self.props.setRenewingSubscription Nothing
```

Comment:
We should probably set `wrapperProgress = AsyncWrapper.Ready` here too

Reply:
That's already invoked by poller's `onComplete`. This whole `onError`/`onSuccess` logic got a bit duplicated in this. But I'll add it, if only for consistency's sake. I may return to clean this up later.